lib/src/blaze/fetch_full_transaction.rs

```diff
@@ -543,11 +558,83 @@ impl FetchFullTxns {
             }
         }
     }
-    async fn scan_orchard_bundle(keys: &Arc<RwLock<Keys>>) {
-        //Todo: Implement this!
-        let fvks = keys
-            .read()
-            .await
-            .get_all_orchard_keys_of_type::<orchard::keys::FullViewingKey>();
+    async fn scan_orchard_bundle(
+        config: &LightClientConfig,
+        transaction: &Transaction,
+        height: BlockHeight,
+        unconfirmed: bool,
+        block_time: u32,
+        keys: &Arc<RwLock<Keys>>,
+        wallet_transactions: &Arc<RwLock<WalletTxns>>,
+        is_outgoing_transaction: &mut bool,
+        outgoing_metadatas: &mut Vec<OutgoingTxMetadata>,
+    ) {
+        scan_bundle(
+            config,
+            transaction,
+            height,
+            unconfirmed,
+            block_time,
+            keys,
+            wallet_transactions,
+            is_outgoing_transaction,
+            outgoing_metadatas,
```

Comment:
Oh...   I get it!   All the additional args are transformations represented as functions.   Very nice!